generators/generator-constants.js

```diff
@@ -91,8 +91,8 @@ const DOCKER_KAFKA = `confluentinc/cp-kafka:${KAFKA_VERSION}`;
 const DOCKER_ZOOKEEPER = `confluentinc/cp-zookeeper:${KAFKA_VERSION}`;
 const DOCKER_SONAR_VERSION = '9.5.0-community';
 const DOCKER_SONAR = `sonarqube:${DOCKER_SONAR_VERSION}`;
-const DOCKER_CONSUL_VERSION = '1.12.2';
-const DOCKER_CONSUL = `consul:${DOCKER_CONSUL_VERSION}`;
+const DOCKER_CONSUL_VERSION = '1.12.2-debian-11-r12';
```

Comment:
It looks like 1.12.3 is the latest version. https://www.consul.io/downloads

Reply:
Probably released after I started the update. I'll change to that